packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,33 @@
 version: 1
 transforms:
 
+  # Change made in https://github.com/flutter/flutter/pull/20649
+  # TODO(Piinks): Add tests when `bulkApply:false` testing is supported.
```

Comment:
Is there an issue on "dart fix" to add testing support for this that we can link from here?

Reply:
And do we know what the timeline for adding that support is?

Reply:
Filed https://github.com/dart-lang/sdk/issues/44639